Nomic_Analysis/0.umap/SHSY5Y/nbconverted/SH-SY5Y_umap.py

```diff
@@ -20,17 +20,25 @@
 nELISA_plate_430420_SH_SY5Y_path = pathlib.Path(
     "../../Data/clean/Plate2/nELISA_plate_430420_SH_SY5Y.csv"
 )
-manual_cluster_path = pathlib.Path(
-    "../../Data/clean/Plate2/Manual_Treatment_Clusters.csv"
+manual_cluster_1_path = pathlib.Path(
+    "../../Data/clean/Plate2/Manual_Treatment_Clusters_1.csv"
+)
+
+manual_cluster_2_path = pathlib.Path(
+    "../../Data/clean/Plate2/Manual_Treatment_Clusters_2.csv"
 )
 
 
 nELISA_plate_430420_SH_SY5Y = pd.read_csv(nELISA_plate_430420_SH_SY5Y_path)
-manual_clusters = pd.read_csv(manual_cluster_path)
+manual_clusters_1 = pd.read_csv(manual_cluster_1_path)
+manual_clusters_2 = pd.read_csv(manual_cluster_2_path)
 
 nELISA_orgingal_plate = nELISA_plate_430420_SH_SY5Y.copy()
 
 
+# In[ ]:
+
+
 # In[3]:
 
 
```

Comment:
I think Greg prefers that the notebook boxes are evaluated in order before converting to `.py`. If it would be a quick rerun consider rerunning and reconverting Jupyter notebook to increase python file readability.

Reply:
re-running each nb and converting!